operator/builtin/input/file/fingerprint_test.go

```diff
@@ -212,4 +214,49 @@ func TestFingerprintStartsWith(t *testing.T) {
 	}
 }
 
+func TestFingerprintStartsWith_FromFile(t *testing.T) {
+	rand.Seed(112358)
```

Comment:
This global may not work well if tests are run in parallel (do we?).
`r := rand.New(rand.NewSource(112358))` and using `r.Read()` should be safer.

Reply:
Good catch. Fixed.